tiledb/ml/models/pytorch.py

```diff
@@ -96,13 +92,25 @@ def save(
             meta=meta,
         )
 
+        # Summary writer
+        if summary_writer:
+            tb = TensorBoardTileDB(f"{self.uri}-tensorboard", self.ctx, self.namespace)
+            tb.save(log_dir=os.path.join(summary_writer.log_dir), update=update)
+
+            # Create group for first time when callback is activated
+            if not update:
+                tiledb.group_create(f"{self.uri}-group", self.ctx)
+                grp = tiledb.Group(f"{self.uri}-group", mode="w", ctx=self.ctx)
+                grp.add(self.uri)
+                grp.add(f"{self.uri}-tensorboard")
```

Comment:
This logic is duplicated in `TensorflowKerasTileDBModel`

Reply:
True it can become a mangled function in `base_class`. But it would be inhereted in `sklearn` and `tensorboard` as well and there either it is not supported e.g. `sklearn` or no point in existing `tensorboard`

Reply:
It doesn't have to be a method necessarily, it can be a plain helper function.